@navikt/core/css/darkside/toggle-group.darkside.css

```
@@ -1,156 +1,106 @@
 .navds-toggle-group__wrapper {
   display: grid;
   justify-items: start;
-  gap: var(--a-spacing-2);
+  gap: var(--ax-spacing-2);
 }
 
 .navds-toggle-group__wrapper--fill {
   justify-items: initial;
 }
 
+/* Main Togglegroup */
 .navds-toggle-group {
-  border-radius: var(--a-border-radius-medium);
-  background-color: var(--ac-toggle-group-bg, var(--a-surface-transparent));
-  box-shadow: inset 0 0 0 1px var(--ac-toggle-group-border, var(--a-border-default));
-  padding: calc(var(--a-spacing-1) + 0.0625rem);
-  gap: var(--a-spacing-1);
+  border-radius: var(--ax-border-radius-medium);
+  background-color: transparent;
+  box-shadow: inset 0 0 0 1px var(--ax-border-default);
+  padding: var(--ax-spacing-1);
+  gap: var(--ax-spacing-2);
```

```suggestion
  gap: var(--ax-spacing-1);
```
After designreview, we want to test spacing-1 here

---

🙆‍♂️